css-canvas-medical/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-50">
+        <img src="images/logo-blue.svg" class="column-40">
+      </div>
+      <div class="nav column-50">
+        <a href="./additional-examples/product.png">Product</a>
+      </div>
+      <div class="nav">
+        <a href="./additional-examples/implementation.png">Implementation</a>
+      </div>
+      <div class="nav">
+        <a href="./additional-examples/about-us.png">About Us</a>
+      </div>
+      <div class="nav">
+        <a href="./additional-examples/blog.png">Blog</a>
+      </div>
+      <div class="column-15">
+        <button class="contact-us" type="button">Contact Us</button>
+      </div>
```

Comment:
Shouldn't all of these be a child of a `div` with the `column-50` class? I doubt you want just your first link to take up 50% of the available width on its own.